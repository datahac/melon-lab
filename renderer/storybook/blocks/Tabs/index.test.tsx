import React from 'react';
import { TabContent, Tabs } from './index';

const data = {
  activeTabIndex: 0,
  handleTabClick: () => null,
};

describe('Tabs', () => {
  const defaultElement = (
    <Tabs {...data}>
      <TabContent title="Tab 0">Tab 0</TabContent>
      <TabContent title="Tab 1">Tab 1</TabContent>
    </Tabs>
  );
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(defaultElement);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
