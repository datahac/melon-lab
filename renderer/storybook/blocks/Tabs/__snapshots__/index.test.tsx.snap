// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs should render correctly 1`] = `
<div
  className="tabs"
  role="tablist"
>
  <style
    jsx={true}
  >
    <type
      0="p"
      1="r"
      2="o"
      3="p"
      4="s"
      key="key"
    >
      rendered
    </type>
  </style>
  <div
    className="tabs__nav"
  >
    <div
      aria-label="Tab 0"
      aria-selected={true}
      className="tabs__tab tabs__tab--active"
      key="0"
      onClick={[Function]}
      onKeyDown={[Function]}
      onKeyPress={[Function]}
      tabIndex={1}
    >
      Tab 0
    </div>
    <div
      aria-label="Tab 1"
      aria-selected={false}
      className="tabs__tab"
      key="1"
      onClick={[Function]}
      onKeyDown={[Function]}
      onKeyPress={[Function]}
      tabIndex={1}
    >
      Tab 1
    </div>
  </div>
  <div
    className="tabs__content tabs__content--active"
    key="0"
  >
    <TabContent
      title="Tab 0"
    >
      Tab 0
    </TabContent>
  </div>
  <div
    className="tabs__content"
    key="1"
  >
    <TabContent
      title="Tab 1"
    >
      Tab 1
    </TabContent>
  </div>
</div>
`;
